Source/Scene/Expression.js

```diff
@@ -79,10 +79,10 @@ define([
     };
 
     var binaryFunctions = {
-        atan2 : Math.atan2,
-        pow : Math.pow,
-        min : Math.min,
-        max : Math.max
+        atan2 : { evaluate: Math.atan2, requireTypeMatch: true },
+        pow : { evaluate: Math.pow, requireTypeMatch: true },
+        min : { evaluate: Math.min, requireTypeMatch: false },
+        max : { evaluate: Math.max, requireTypeMatch: false }
```

Comment:
While this is more automated, I think its slightly overbuilt considering there are only 4 function. It should be fine to check the call name in `getEvaluateBinaryFunction` and apply the special cases.